README.md

```diff
@@ -16,6 +16,7 @@
 
 * 🇬🇧 English
 * 🇫🇷 French
+* DE German
```

Comment:
Use the DE emoji that can be found here: [deutch/german emoji](https://emojipedia.org/search?q=german)